src/bid/encoding.rs

```diff
@@ -7,13 +7,17 @@
 //! Encoding module for Bid structure.
 //! See: <https://hackmd.io/@7dpNYqjKQGeYC7wMlPxHtQ/BkfS78Y9L>
 
+cfg_if::cfg_if! {
+if #[cfg(feature = "canon")] {
+       use alloc::vec::Vec;
+       use dusk_plonk::constraint_system::ecc::Point as PlonkPoint;
+       use dusk_plonk::prelude::*;
+    }
+}
+
```

Comment:
You don't need `cfg_if` you can use a single `use` statement as we have done before.